sharded_queue/api.lua

```diff
@@ -319,24 +293,30 @@ function sharded_queue.statistics(tube_name)
             leader_only = true
         })
 
-    local ok, stats_collection = map('tube_statistic',
-        { tube_name = tube_name }, storages)
+    local stats_collection, err = cartridge_pool.map_call('tube_statistic',
+        {{ tube_name = tube_name }}, {uri_list=storages})
 
-    if not ok or stats_collection == nil then
+    if err then
+        error(err)
+    end
+
+    if fun.iter(stats_collection):length() == 0 then
         log.info(stats_collection)
-        return
+        return nil
     end
+
     -- merge
-    local stat = stats_collection[1]
-    for i = 2, #stats_collection do
-        for name, count in pairs(stats_collection[i].tasks) do
+    local _, stat = fun.head(stats_collection)
+    fun.tail(stats_collection):each(function(_, uri_stat)
```

Comment:
1. This changes is irrelevant for this patch
1. Please, try to avoid using buggy luafun (e.g. https://github.com/tarantool/tarantool/issues/4252#issuecomment-528334600)

Reply:
Fixed https://github.com/tarantool/sharded-queue/pull/19/commits/48cc11c6a319e8a33a0281979f65a3942f309f0d